docs/web3.main.rst

```diff
@@ -161,16 +161,16 @@ Encoding and Decoding Helpers
         >>> Web3.toInt(hexstr='000F')
         15
 
-.. py:method:: Web3.toJSON(obj)
```

Comment:
Might be worth adding a note in the docs here that `toJSON` is deprecated, and `to_json` is preferred now.

Reply:
Agreed. I think keeping this doc, since the method still exists, and adding a warning might be best.

```rst
.. py:method:: Web3.toJSON(obj)
  
    .. warning:: Deprecated: This method is deprecated in favor of :class:`~to_json`

Reply:
done!